helm/sematic-server/values.yaml

```diff
@@ -8,6 +8,14 @@ aws:
   storage_bucket: my-s3-bucket # REPLACE ME
   bucket_region: my-s3-region # REPLACE ME
 
+gcp:
```

Comment:
As you suggested, I broke it up such that instead of enabling/disabling gcp wholesale, you enable/disable storage or service account distinctly. So I removed the overall "gcp" enabled/disabled setting. As far as ensuring that only one storage option is enabled at a time, it's set up so that if you have multiple enabled, one will take precedence and "win," which is the same as is done here.